apps/condo/domains/meter/tasks/deleteReadingsOfDeletedMeter.js

```diff
@@ -1,22 +1,25 @@
 const { getSchemaCtx, find } = require('@open-condo/keystone/schema')
 const { createTask } = require('@open-condo/keystone/tasks')
 
-const { MeterReading } = require('../utils/serverSchema')
+const { MeterReading } = require('@condo/domains/meter/utils/serverSchema')
+const { PropertyMeterReading } = require('@condo/domains/meter/utils/serverSchema')
 
 /**
  * Soft delete meter readings after soft delete meter
  */
-async function deleteReadingsOfDeletedMeter (deletedMeter, deletedMeterAt) {
+async function deleteReadingsOfDeletedMeter (deletedMeter, deletedMeterAt, isPropertyMeter = false) {
     const { keystone: context } = await getSchemaCtx('Property')
 
     const meterId = deletedMeter.id
-    const meterReadings = await find('MeterReading', {
+    const meterReadings = await find(isPropertyMeter ? 'PropertyMeterReading' : 'MeterReading', {
```

Comment:
If you can you use `__typename` field of the `deletedMeter` schema record object, argument `isPropertyMeter` will be redundant.
```suggestion
    const meterReadings = await find(deletedMeter.__typename === 'PropertyMeter' ? 'PropertyMeterReading' : 'MeterReading', {
```